companion/src/firmwares/boards.cpp

```diff
@@ -96,6 +96,8 @@ uint32_t Boards::getFourCC(Type board)
       return 0x4778746F;
     case BOARD_RADIOMASTER_T8:
       return 0x4378746F;
+    case BOARD_iFlight_Commando8:
```

Comment:
all UPPER, I stop here, I think you got the idea

Reply:
code cleaned up